recipe/meta.yaml

```diff
@@ -1,23 +1,18 @@
-{% set version = "4.0.9" %}
+{% set version = "4.0.10" %}
 
 package:
   name: libtiff
   version: {{ version }}
 
 source:
   url: http://download.osgeo.org/libtiff/tiff-{{ version }}.tar.gz
-  sha256: 6e7bdeec2c310734e734d19aae3a71ebe37a4d842e0e23dbb1b8921c0026cfcd
+  sha256: 2c52d11ccaf767457db0c46795d9c7d1a8d8f76f68b0b800a3dfe45786b996e4
   patches:
-    - 0001-CVE-2017-9935.patch
-    - 0002-CVE-2017-18013.patch
-    - 0003-CVE-2018-5784.patch
-    - 0004-CVE-2017-11613_part1.patch
-    - 0005-CVE-2017-11613_part2.patch
-    - 0006-CVE-2018-7456.patch
-    - 0007-CVE-2017-17095.patch
+    - patches/fix_TIFFReadRawStrip_man_page_typo.patch
+    - patches/fixed_lossless_webp_compression_config.patch
```

Comment:
@mingwandroid I found a debian package with these two patches.
I believe they dropped the rest.